quickstep/src/com/android/launcher3/uioverrides/QuickstepLauncher.java

```diff
@@ -1049,14 +1049,11 @@ public ActivityOptionsWrapper getActivityLaunchOptions(View v, @Nullable ItemInf
     @Override
     public ActivityOptionsWrapper makeDefaultActivityOptions(int splashScreenStyle) {
         RunnableList callbacks = new RunnableList();
-        ActivityOptions options = LawnchairQuickstepCompat.ATLEAST_R ? ActivityOptions.makeCustomAnimation(
-                this, 0, 0, Color.TRANSPARENT,
-                Executors.MAIN_EXECUTOR.getHandler(), null,
-                elapsedRealTime -> callbacks.executeAllAndDestroy()) : ActivityOptions.makeBasic();
-        if (LawnchairQuickstepCompat.ATLEAST_T) {
-            options.setSplashScreenStyle(splashScreenStyle);
-        }
-        Utilities.allowBGLaunch(options);
+        ActivityOptions options = ActivityOptions.makeCustomAnimation(
+            this, 0, 0, Color.TRANSPARENT,
+            Executors.MAIN_EXECUTOR.getHandler(), null,
+            elapsedRealTime -> callbacks.executeAllAndDestroy());
+        options.setSplashScreenStyle(splashScreenStyle);
```

Comment:
Reverted this logic back in Launcher3.

Reply:
We override `makeDefaultActivityOptions` in `LawnchairLauncher` and don't call `super`, so it should be safe here.

Reply:
Yeah, you're right